java/src/main/java/ai/rapids/cudf/HostColumnVector.java

```diff
@@ -1085,9 +1120,11 @@ private void appendChildOrNull(ColumnBuilder childBuilder, Object listElement) {
       } else if (listElement instanceof BigDecimal) {
         childBuilder.append((BigDecimal) listElement);
       } else if (listElement instanceof List) {
-        childBuilder.append((List) listElement);
+        childBuilder.append((List<?>) listElement);
```

Comment:
Why this is needed?

Reply:
Just to eliminate the warn message about 'unchecked type' from IDE.